experiments/AtmosLES/convective_bl_model.jl

```diff
@@ -0,0 +1,322 @@
+#!/usr/bin/env julia --project
+# This experiment file establishes the initial conditions, boundary conditions,
+# source terms and simulation parameters (domain size + resolution) for the
+# GABLS LES case (Beare et al, 2006; Kosovic and Curry, 2000).
+
+## ### Convective Boundary Layer LES
+## @article{Kitamura2016,
+## author={Kitamura, Yuji
+## and Ito, Junshi},
+## title={Revisiting the Bulk Relation for Heat Flux in the Free Convection Limit},
+## journal={Boundary-Layer Meteorology},
+## year={2016},
+## month={Jan},
+## day={01},
+## volume={158},
+## number={1},
+## pages={93-103},
+## issn={1573-1472},
+## doi={10.1007/s10546-015-0075-z},
+## url={https://doi.org/10.1007/s10546-015-0075-z}
+## }
+#
+# To simulate the experiment, type in
+#
+# julia --project experiments/AtmosLES/stable_bl_les.jl
```

Comment:
This should be changed to the convective bl

Reply:
done